server.js

```diff
@@ -53,11 +53,4 @@ app.use(function(req, res, next) {
 
   app.listen(port, function () {
 
-    console.log("Server listening on: http://localhost:" + port);
```

Comment:
@MCabral773 I accidentally deleted lines 56/67(end of app.list function) here when I was accepting the merge conflicts. Would you mind adding them back and I'll approve? Sorry for the inconvenience!